### ISSUE TITLE
Missing character (is a square)

### ISSUE THREAD

**Issue by octoshrimpy** [jgthms/bulma#1332]
This is about **the Docs**.

### Overview of the problem

This is about the Bulma **Docs** 

My **browser** is: 
Chrome  61.0.3163.100 (Official Build) (64-bit)

### Description
Missing icon/emoji from the `/love` page of the site. 
https://bulma.io/love/

It only happens in chrome as far as I can tell. Firefox shows the image just fine (see below)

### Steps to Reproduce
1. Navigate to https://bulma.io/love/ in Chrome

### Expected behavior

this is in firefox:
![image](https://user-images.githubusercontent.com/7097172/31732014-37734286-b3f4-11e7-87d7-bf0e6c62c442.png)

### Actual behavior

![image](https://user-images.githubusercontent.com/7097172/31732020-400fedb8-b3f4-11e7-856c-a5fa0f38bdb5.png)

**Comment by octoshrimpy** [comment]
Updated to Chrome 62.0.3202.62 (Official Build) (64-bit) LTS and still shows the same square.

**Comment by SimplyThomas** [comment]
I'm not seeing the issue on OSX Chrome Version 62.0.3202.62 (Official Build) (64-bit)

Do you have an ad-blocker or something that could be interfering with loading the emoji?

**Comment by octoshrimpy** [comment]
@SimplyThomas just tried it in incognito, an without adblocking (ublock origin), also turned off any tampermonkey extensions that might be interfering. all yielded the same results: square is still there. 
I am on windows 10 pro.

Tested on coworker's machine, also win10 pro, and chrome: square.

**Comment by octoshrimpy** [comment]
Might be worth looking into this: https://ellekasai.github.io/twemoji-awesome/ seems like something that Bulma could benefit from. :)

**Comment by SimplyThomas** [comment]
@octoshrimpy I jumped on my Win 10 Pro machine with Chrome Version 61.0.3163.100 (Official Build) (64-bit) and I'm still not able to replicate your results. Are any of the emojis on that page displaying for you? There's more than just this one and I'm still able to display all of them without issue.

![bulma](https://user-images.githubusercontent.com/6202961/31797215-7aa1124c-b52d-11e7-9f38-0a241236d4f2.PNG)

**Comment by octoshrimpy** [comment]
@SimplyThomas yep! it's just that one. not sure what is going on. I'll test it at home, when outside the university's wifi. Here's what it looks like to me: 
![image](https://user-images.githubusercontent.com/7097172/31797934-28eeb5ec-b4ee-11e7-941e-0540348b1f8b.png)

**Comment by ctolkien** [comment]
What operating system are you on @octoshrimpy ?

**Comment by octoshrimpy** [comment]
@ctolkien win10 Pro

**Comment by ctolkien** [comment]
@octoshrimpy and which version? ('winver' at the run command dialog).

**Comment by octoshrimpy** [comment]
@ctolkien @SimplyThomas My mistake, not Pro, but enterprise. 
![image](https://user-images.githubusercontent.com/7097172/31835110-92cfd172-b58d-11e7-85fa-c295c09f7ecd.png)

**Comment by ctolkien** [comment]
@octoshrimpy  - you're several versions behind the latest Win10 and as of this month 1511 is entirely unsupported by MSFT.

Wider emoji support is added in later versions of Win10.

After 1511, we've had 1607, 1703 and recently 1709 as "major" feature updates.

**Comment by octoshrimpy** [comment]
@ctolkien just checked, cannot upgrade as it is a work machine, and the creators update breaks some IIS settings we need. 

seems like the bug was found (and it was on my side), this issue can be closed.

**Comment by jnsdrtlf** [comment]
For compatibility, it might be good to add the emojis as SVGs instead of unicode characters. Every system displays them differently - or not at all.
Examples:
 - http://twitter.github.io/twemoji/
 - https://afeld.github.io/emoji-css/

**Comment by stavlocker** [comment]
@jnsdrtlf While these are two cool libraries, I don't think it should be included in Bulma. It's a CSS framework, and it shouldn't be the one providing the emojis for the developers. It's a personal decision for the developer if they want custom emojis or anything of the sort.
@jgthms:

> seems like the bug was found (and it was on my side), this issue can be closed.

**Comment by jnsdrtlf** [comment]
>  I don't think it should be included in Bulma

No, they shouldn't. I totally agree with you. That would be a stupid idea 😄. I was only talking about the docs and the website. Maybe the emojis should be replaced. But I'm not sure if emojis are really that important.

**Comment by octoshrimpy** [comment]
I second @jnsdrtlf. The icon fonts would only be used for the docs + website, not included in the bulma framework itself.
